webpay/pay/tests/test_tasks.py

```diff
@@ -87,7 +96,8 @@ def req_ok(req):
             eq_(dd['typ'], payload['typ'])
             eq_(dd['response']['price']['amount'], 1)
             eq_(dd['response']['price']['currency'], u'USD')
-            jwt.decode(req['notice'], 'f', verify=True)
+            jwt.decode(req['notice'], 'f', verify=True,
+                       audience=self.payment_issuer)
```

Comment:
I think as long as it's different from the default than it should be pretty robust